src/api/app/controllers/webui/users/beta_features_controller.rb

```diff
@@ -4,6 +4,10 @@ class Webui::Users::BetaFeaturesController < Webui::WebuiController
   def index
     @disabled_beta_features = policy_scope(DisabledBetaFeature).pluck(:name)
     @user = User.session!
+
+    # Rolled-out beta feature cannot be switched, so we do not display them to users
+    rolled_out_beta_features = Flipper::Adapters::ActiveRecord::Gate.where(key: 'groups', value: 'rollout').pluck(:feature_key)
```

Comment:
I know in production we usually go through the "rollout way" to enable the feature to everyone... But we could also take into account features that are "Fully Enabled", in case someone goes that way (something that happens frequently in the development environment).

So, apart from rejecting `Flipper::Adapters::ActiveRecord::Gate.where(key: 'groups', value: 'rollout')` you can also reject `Flipper::Adapters::ActiveRecord::Gate.where(key: 'boolean', value: 'true')`.

What do you think?

Reply:
Good point, I'll do that.

Reply:
It's there now.